src/iri/utils.ts

```diff
@@ -20,12 +21,15 @@ export function isIRI( iri:string ):boolean {
 	return hasProtocol( iri ) || ! isAbsolute( iri );
 }
 
+export function isBNodeLabel( label:string ):boolean {
+	return /^_:/.test( label );
+}
 
 let prefixRegex:RegExp = /([A-Za-z](([A-Za-z_\-0-9]|\.)*[A-Za-z_\-0-9])?)?:/;
 let prefixNormalizeRegex:RegExp = /([_~.\-!$&'|()*+,;=/?#@%])/g;
 
 export function isPrefixed( iri:string ):boolean {
-	return ! ! iri.match( prefixRegex ) && ! hasProtocol( iri );
+	return /^(?!_:)[^]*?:/.test( iri ) && ! hasProtocol( iri );
```

Comment:
Same ^^

Reply:
Also, could you explain why the reason behind the change of the expression being used?

Reply:
I changed the regex to a soft one, so the carbonldp SKD could validate and remove invalid characters when the IRI is a possible prefixed name.